src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -69,15 +78,16 @@ private static void BuildTocMap(Context context, Document fileToBuild, TableOfCo
             List<TableOfContentsItem> tocItems,
             JObject metadata,
             List<Document> referencedDocuments,
-            List<Document> referencedTocs)
+            List<Document> referencedTocs,
+            List<string> monikers)
 
-            Load(Context context, Document fileToBuild, DependencyMapBuilder dependencyMapBuilder = null)
+            Load(Context context, Document fileToBuild, DependencyMapBuilder dependencyMapBuilder = null, Dictionary<Document, List<string>> monikersMap = null)
```

Comment:
should `monikersMap` and `dependencyMapBuilder` always not null?

Reply:
when build toc map, they should be null

---
In reply to: [235841942](https://github.com/dotnet/docfx/pull/3655#discussion_r235841942) [](ancestors = 235841942)